less/v2/high-contrast/chat.less

```
@@ -0,0 +1,3 @@
+.dnd5e-flag-high-contrast {
+  .dice-result .dice-total > i { opacity: unset; }
+}
```

We should also increase the contrast for the AC indicator which is pretty low:

```suggestion
  :is(.chat-popout, #chat-log) .message .dnd5e2.evaluation .target.hit .ac { opacity: .8; }
}
```